src/assets/wise5/components/common/feedbackRule/feedback-rule-help/feedback-rule-help.component.ts

```diff
@@ -0,0 +1,7 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
The OnInit import can be removed since it isn't used.

Reply:
thanks, removed.